_data/hosting.yml

```diff
@@ -100,6 +100,14 @@ websites:
       software: Yes
       doc: https://blog.nearlyfreespeech.net/2014/02/28/price-cuts-more-security-and-recovery-options/
 
+    - name: Online
+      url: https://www.online.net/
+      img: online.png
+      tfa: Yes
+      sms: Yes
+      software: Yes
+      twitter: online_en
```

Comment:
they support tfa right? Then you don't need a twitter handle. For more info see [our wiki](https://github.com/jdavis/twofactorauth/wiki/FAQ_New_TFA_SITE)